kubernetes/wes-node-influxdb.yaml

```diff
@@ -34,6 +34,21 @@ spec:
         containers:
           - image: influxdb:2.1.1
             name: wes-node-influxdb
+            env:
+            - name: DOCKER_INFLUXDB_INIT_MODE
+              value: "setup"
+            - name: DOCKER_INFLUXDB_INIT_USERNAME
+              value: "waggle"
+            - name: DOCKER_INFLUXDB_INIT_PASSWORD
+              value: "wagglewaggle"
+            - name: DOCKER_INFLUXDB_INIT_ORG
+              value: "waggle"
+            - name: DOCKER_INFLUXDB_INIT_BUCKET
+              value: "waggle"
+            - name: DOCKER_INFLUXDB_INIT_RETENTION
+              value: "7d"
+            - name: INFLUX_CONFIGS_PATH
+              value: /var/lib/influxdb2/influx-configs
```

Comment:
This looks great! I think it'll make things much easier.

I noticed I had a couple more for INFLUXD_BOLT_PATH and INFLUXD_ENGINE_PATH: https://github.com/waggle-sensor/honeyhouse-config/blob/4dc4fb9fd5ae8412339360da401de016c3424a22/beehives/beehive-dev/influxdb/statefulset.yaml#L17

I think that may have just been me being paranoid... the defaults are probably fine.

Reply:
Yeah, I thought having them being default would be fine.